### PR TITLE
fix(anchor-links): Remove duplicate anchor links

### PR DESCRIPTION
**Pull request by leevye**
The docs site uses an extension that will automatically make any header into an anchor link. 

If there more than one header with the same text (aka multiple "Endpoint" headers) on the same page, the anchor link will only point to the first one

To avoid this, and improve SEO by not having too many headers, I have changed every page where "Endpoint" and "Response" appear multiple times to bolded words vs headers

**Endpoint** vs

### CODE REVIEW

File: app/plugins/aws-lambda.md [hunk 1]
```diff
@@ -128,7 +128,7 @@ When the plugin is added to an API entity (which is deprecated as of 0.13.0),
 it is the `upsream_url` property which must be specified and resolvable as well
 (but ignored).
 
-#### Response plugins
+**Response** plugins
```
Comment (coopr) [inline app/plugins/aws-lambda.md]: This update needs to be made in the new `app/_hub/kong-inc/aws-lambda/index.md` file.

Comment (leevye) [reply app/plugins/aws-lambda.md]: Actually #### Response plugins should be fine - I will actually reverse this one

**Comment by leevye** [comment]
I searched for every instance of ####Endpoint and ####Response and swapped them, so we should be GTG

**Comment by coopr** [comment]
Still some `#### Endpoint` in the docs - https://github.com/Kong/docs.konghq.com/search?q=%23%23%23%23+Endpoint&unscoped_q=%23%23%23%23+Endpoint

**Pull request merged**